nemo/collections/tts/modules/common.py

```diff
@@ -219,7 +219,7 @@ def conv_to_padded_tensor(self, context: Tensor, lens: Tensor) -> Tensor:
         ret = torch.nn.utils.rnn.pad_sequence(context_embedded, batch_first=True)
         return ret
 
-    @torch.jit.export
+    @torch.jit.ignore
     def masked_conv_to_sequence(self, context: Tensor, lens: Tensor, enforce_sorted: bool = False) -> PackedSequence:
```

Comment:
Ignore ? Mask calculation will be incorrect then in export mode

Reply:
This function is never used (see ADLR comment below). I do plan to make it work - but with current scripting changes, it breaks as it gets scripted anyway. Let me remove the qualifier altogether to avoid confusion.

Reply:
Ok